frontend/src/app/Settings/Facility/Components/ManageDevices.tsx

```diff
@@ -16,6 +18,41 @@ interface Props {
   onChange: (selectedItems: string[]) => void;
 }
 
+type SupportedDisease = {
```

Comment:
the prop type was typed as any, so adding a utility type here so TS doesn't yell at me. If anyone knows of an existing type that I can steal instead, lmk.

Happy also to add a tech debt ticket to fix this particular typing inconsistency frontend-wide if folks think it's worth it.